bigframes/session/__init__.py

```diff
@@ -332,13 +333,15 @@ def session_id(self):
     @property
     def objects(
         self,
-    ) -> collections.abc.Set[
+    ) -> Tuple[
```

Comment:
Technically a breaking change. Maybe OK since we didn't actually document this property, but might be better to change from `Set` to a broader type like `Iterable`.

Reply:
Hmm, yeah, actually, should we just make this private? Added this very recently and wasn't really intending this for user consumption.